mne/io/curry/curry.py

```diff
@@ -542,7 +542,7 @@ def _read_annotations_curry(fname, sfreq="auto"):
 
 
 @verbose
-def read_raw_curry(fname, preload=False, verbose=None):
+def read_raw_curry(fname, preload=False, verbose=None) -> "RawCurry":
```

Comment:
question from a type-hints n00b: why is `"RawCurry"` a string here? Some of the other type hints like `RawEDF` are not strings, but references to the Class object.

Reply:
See my top comment:

"I had to annotate via strings (names of the classes) in many places because the reader functions appeared before the actual class definition, i.e. I had to make a forward reference."

It's ugly, but I wanted to avoid having to re-order the code. 

Thoughts on this?

Reply:
Yes. Rearrange the code 😄.

Reply:
oops, sorry!  I did read the PR description but somehow my eyes skipped right over that sentence.  Let's leave it as-is for now, someone can do the copy-paste reordering later in a commit that we can add to `.git-blame-ignore-revs`